docusaurus.config.js

```diff
@@ -393,6 +393,16 @@ module.exports = {
                 sidebarPath: false,
             }),
         ],
+        [
+          "content-docs",
```

Comment:
If it's a standalone page, we may not need a docs plugin instance.

Reply:
it's a simplified way to implement the below effect (the absolute same TOC effect with the docs page)

<img width="156" alt="image" src="https://user-images.githubusercontent.com/76024046/235055009-3c5a529b-1463-42c0-ad38-0605b597694d.png">

@tisonkun

Reply:
Thanks for your explanation! I agree with this trade-off now :D